arbitrum-helpers/networks/RetryProvider.ts

```diff
@@ -18,6 +18,11 @@ export class RetryProvider extends providers.JsonRpcProvider {
   public async perform(method: string, params: any): Promise<any> {
     let attempt = 0
 
+    // do not retry txs
+    if (method === 'eth_sendRawTransaction' || method === 'sendTransaction') {
```

Comment:
Given this check, it's no longer necessary to check if `method === 'sendTransaction'` in `handleError()` on L48.